examples/114_allops.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Konstruktion.cdy</title>
+    <style type="text/css">
+        * {
+            margin: 0px;
+            padding: 0px;
+        }
+
+        #CSConsole {
+            background-color: #FAFAFA;
+            border-top: 1px solid #333333;
+            bottom: 0px;
+            height: 200px;
+            overflow-y: scroll;
+            position: fixed;
+            width: 100%;
+        }
+    </style>
+    <script type="text/javascript" src="../build/js/Cindy.js"></script>
+
+<script id='csmove' type='text/x-cindyscript'>
+println("allpoints, should be A, B, C, D, E, F, G, H, K, L, M, N, O");
+println(allpoints());
+println("alllines, should be a,b,c");
+println(alllines());
+println("allsegments, should be a,c");
+println(allsegments());
+println("allconics, should be C1, C2, C0");
+println(allconics());
+println("allcircles, should be C0, C2");
+println(allcircles());
+println("allmasses, should be M, O");
+println(allmasses());
+println("allsprings, should be c");
+println(allsprings());
+
+println("allelements");
+println(allelements());
+
+
+// INCIDENCES
+
+println("allpoints incident to b, should be K");
+println(allpoints(b));
+println("allpoints incident to C1, should be A B C D E");
+println(allpoints(C1));
+println("alllines incident to K, should be b");
+println(alllines(K));
+
+println("allsegments incident to L, should be c");
+println(allsegments(L));
```

Comment:
`alllines()` and `alsegments()` correctly list `no` even though you don't include it in the expected result.